src/main/java/racinggame/domain/Cars.java

```diff
@@ -0,0 +1,37 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Cars {
+
+    private static final Integer START_INDEX = 0;
+    private static final Integer MAX_MOVE_VALUE = 9;
+
+    private final List<Car> cars;
+    private final Random random;
+
+    public Cars(Integer carCount) {
+        random = new Random();
+        cars = IntStream.range(START_INDEX, carCount)
+                .mapToObj((idx) -> new Car())
+                .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
```suggestion
               .collect(Collectors.toList());
```
위 코드가 대신할 수 있을것 같아요 😄